org.librarysimplified.r2.demo/src/main/res/layout/demo_activity_host.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+  android:layout_width="match_parent"
+  android:layout_height="match_parent"
+  android:orientation="vertical">
+
+  <androidx.appcompat.widget.Toolbar
+    android:id="@+id/mainToolbar"
+    android:theme="?android:attr/actionBarTheme"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content" />
+
+  <FrameLayout
+    android:id="@+id/mainFragmentHolder"
+    android:layout_width="match_parent"
+    android:layout_height="0dp"
+    android:layout_weight="1" />
```

Comment:
There's no need of this. Since it's the last element on the LinearLayout, you can set its height to _match_parent_

Reply:
Thanks. This came from upstream, so I'm going to leave it for now so I don't make any conflicts.